PatientWebApplication/UserMicroserviceApi/Model/EmployeeUser.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace UserMicroserviceApi.Model
+{
+    public class EmployeeUser : User
+    {
+        public double salary { get; set; }
+
+        public string city { get; set; }
+
+        public string email { get; set; }
+        public string password { get; set; }
```

Comment:
Change properties names to start with upper letter and apply in rest of code (remove 'this' from constructors).